pertpy/tools/_coda/_tasccoda.py

```diff
@@ -120,8 +120,10 @@ def load(
                 covariate_df=covariate_df,
             )
             mdata = MuData({modality_key_1: adata, modality_key_2: adata_coda})
-        else:
+        elif type == "sample_level":
             mdata = MuData({modality_key_1: AnnData(), modality_key_2: adata})
+        else:
+            raise ValueError(f'{type} is not a supported type, please refer to "cell_level" or "sample_level" type')
```

Comment:
```suggestion
            raise ValueError(f'{type} is not a supported type, expected "cell_level" or "sample_level"')
```

Maybe like this? "please refer to" reads a bit weird to me.